Sources/StreamChat/Database/DatabaseSession.swift

```diff
@@ -303,6 +305,26 @@ protocol ChannelDatabaseSession {
     func removeChannels(cids: Set<ChannelId>)
 }
 
+extension ChannelDatabaseSession {
```

Comment:
Without it, there are so many errors because of missing the `offset` argument (most of it is in tests). If we want to avoid this extension, I can take the time to fill it in.

Reply:
can't we provide a default value for the offset?

Reply:
or maybe move these to the test target?

Reply:
Unfortunately can't set a default value in protocol but moving the extension to test target is a good idea.

Reply:
Moved it